main/lsp/LSPPreprocessor.cc

```diff
@@ -356,7 +356,7 @@ void LSPPreprocessor::canonicalizeEdits(u4 v, unique_ptr<DidChangeTextDocumentPa
     updates.versionStart = v;
     updates.versionEnd = v;
     string_view uri = changeParams->textDocument->uri;
-    if (config->isUriInWorkspace(uri)) {
+    if (config->isUriInWorkspace(uri) && !config->isSorbetUri(uri)) {
```

Comment:
This is a little awkward, does `isSorbetUri` need to be included in `isUriInWorkspace`?

Reply:
Good question. There's one callsite of `isUriInWorkspace` that assume that it accepts `sorbet:` URIs. I think it might be cleaner to change that callsite instead of these ones!

I'll fix that up before submitting.